src/Illuminate/Database/Query/Builder.php

```diff
@@ -2577,7 +2577,7 @@ protected function enforceOrderBy()
     }
 
     /**
-     * Get an array with the values of a given column.
+     * Get a Collection Instance with the values of a given column.
```

Comment:
why is Instance with a capital I? it's not a proper noun nor a class name

Reply:
Oh yes, it should be "instance". It has been corrected now. Thanks!